### ISSUE TITLE
Infix operators can not be used along with evancz/url-parser

### ISSUE THREAD

**Issue by bardt** [nonpop/elm-purl#2]
## Steps to reproduce

1. Install both [evancz/url-parser](http://package.elm-lang.org/packages/evancz/url-parser/2.0.1) and this package.
2. Import `Url` module and use infix operators, like `</>` or `<?>` in a chain. 
3. Try to compile at least twice.
3. The operators' application order is broken.

[Example](https://ellie-app.com/9hFScNDfga1/1) (compile at least twice)

## Reason

[Infix operators](http://elm-lang.org/docs/syntax#infix-operators) precedence is set in `url-parser` package and is different from the default value. Currently, precedence is applied globally ([related issue](https://github.com/elm-lang/elm-compiler/issues/1096)).

## Suggestions

Unfortunately, we can do nothing about it. If we specify infix precedence explicitly, it will conflict with `url-parser` and will not compile at all.

The only suggestion is to mention this problem in the README.md and provide another example using `append` and `appendParam` methods instead.

**Comment by nonpop** [comment]
Yes, I'm aware of this. I agree this should be mentioned in the readme, and will put it there soon. Another workaround, which I have used, is to use explicit parentheses, as in `(root </> s "foo") </> s "bar"`. It's a little bit ugly but works. Or the infix operators could be changed to something else. I also tried to implement url parsing in this package so I wouldn't need evancz/url-parser at all, but without a record-extension primitive I didn't get the kind of API I wanted.
